modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/resources/META-INF/resources/view_factory_instances.jsp

```diff
@@ -31,7 +31,7 @@ ConfigurationModel configurationModel = (ConfigurationModel)request.getAttribute
 String category = configurationModel.getCategory();
 
 portletDisplay.setShowBackIcon(true);
-portletDisplay.setURLBack(portletURL.toString());
+portletDisplay.setURLBack(redirect);
```

Comment:
This is fine as long as we default "redirect" to portletURL.toString() if it's not been provided.